tests/onnx/test_statistics_aggregator.py

```diff
@@ -12,90 +12,63 @@
-    statistics_points = StatisticPointsContainer()
-    if test_parameters.range_type == RangeType.MINMAX:
-        tensor_collector = ONNXMinMaxStatisticCollector(test_parameters.use_abs_max, test_parameters.reduction_shape,
-                                                        num_samples=len(DATASET_SAMPLES))
-    if test_parameters.range_type == RangeType.MEAN_MINMAX:
-        tensor_collector = ONNXMeanMinMaxStatisticCollector(False, test_parameters.use_abs_max,
-                                                            test_parameters.reduction_shape,
-                                                            num_samples=len(DATASET_SAMPLES))
-    target_node_name = 'Identity'
-    algorithm_name = 'TestAlgo'
-    statistic_point_type = TargetType.POST_LAYER_OPERATION
-    target_point = ONNXTargetPoint(statistic_point_type, target_node_name, 0)
-    statistics_points.add_statistic_point(StatisticPoint(target_point=target_point,
-                                                         tensor_collector=tensor_collector,
-                                                         algorithm=algorithm_name))
-    statistics_aggregator.register_stastistic_points(statistics_points)
-    statistics_aggregator.collect_statistics(model)
-
-    def filter_func(point):
-        return algorithm_name in point.algorithm_to_tensor_collectors and \
-               point.target_point.type == statistic_point_type
-
-    for tensor_collector in statistics_points.get_algo_statistics_for_node(
-            target_node_name,
-            filter_func,
-            algorithm_name):
-        stat = tensor_collector.get_statistics()
-        assert np.allclose(stat.max_values, test_parameters.ref_max_val)
-        assert np.allclose(stat.min_values, test_parameters.ref_min_val)
+
+class TestStatisticsAggregator(TemplateTestStatisticsAggregator):
+    def get_algo_backend_cls(self) -> ONNXMinMaxAlgoBackend:
+        return ONNXMinMaxAlgoBackend
+
+    def get_backend_model(self, dataset_samples):
+        conv_w = self.dataset_samples_to_conv_w(dataset_samples[0])
+        return IdentityConvolutionalModel(input_shape=[1] + INPUT_SHAPE,
+                                          inp_ch=3,
+                                          out_ch=3,
+                                          kernel_size= 3,
+                                          conv_w=conv_w).onnx_model
+
+    def get_statistics_aggregator(self, dataset):
+        return ONNXStatisticsAggregator(dataset)
+
+    def get_dataset(self, samples):
+        def transform_fn(data_item):
+            inputs = data_item
+            return {INPUT_NAME: [inputs]}
+
+        return Dataset(samples, transform_fn)
+
+    def get_target_point(self, target_type: TargetType):
+        target_node_name = IDENTITY_NODE_NAME
+        if target_type == TargetType.OPERATION_WITH_WEIGHTS:
+            target_node_name = CONV_NODE_NAME
+        return ONNXTargetPoint(target_type, target_node_name, 0)
```

Comment:
0 only for activations, not for weights

Reply:
Actually, port_id is never used in onnx backend for now, either for weights nor activations

Reply:
Done, I've added description for the `ONNXTargetPoint`